user-guide/src/syntax.md

```diff
@@ -70,9 +71,11 @@ forall(|x: isize, y: isize| ...)
 The syntax of quantifiers is:
 
 ```plain
-forall(|<bound variable>: <bound variable type>, ...| <filter> ==> <expression>)
+<quantifier_type>(|<bound variable>: <bound variable type>, ...| <filter> ==> <expression>)
```

Comment:
`<filter> ==>` does not make so much sense for existentials. Maybe just show two different versions?

Reply:
A fair point.